react.js

```diff
@@ -61,6 +61,8 @@ module.exports = {
     'react/jsx-uses-vars': 'warn',
     // Prevent missing parentheses around multilines JSX
     'react/jsx-wrap-multilines': 'warn',
+    //  Prevent using Array index in key props
+    'react/no-array-index-key': 'warn',
```

Comment:
I think it's valid when spiking/trying to get something working, but I don't think there's a good use case for keeping it in the code long-term.